tests/aws/services/s3/test_s3.py

```diff
@@ -4799,10 +4808,15 @@ def test_s3_sse_validate_kms_key(
         data = b"test-sse"
         bucket_name = f"bucket-test-kms-{short_uid()}"
         s3_create_bucket(
-            Bucket=bucket_name, CreateBucketConfiguration={"LocationConstraint": "us-west-2"}
+            Bucket=bucket_name,
+            CreateBucketConfiguration={
+                "LocationConstraint": "us-west-2"
+                if TEST_AWS_REGION_NAME != "us-west-2"
+                else "me-south-1"
+            },
         )
```

Comment:
it seems all of these changes conflict with #9570? Should they be removed?

Reply:
yes @bentsku, I will rebase this PR once [#9570](https://github.com/localstack/localstack/pull/9570) is green.